common/PaperSizeProcessor.hh

```diff
@@ -0,0 +1,129 @@
+/* -*- Mode: C++; indent-tabs-mode: t; c-basic-offset: 4; tab-width: 4 -*-  */
+/*
+ * Acquirer.cc
+ * Copyright (C) 2017 Alexander Zaitsev <zamazan4ik@tut.by>
+ *
+ * gImageReader is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * gImageReader is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef GIMAGEREADER_PAPERSIZE_H
+#define GIMAGEREADER_PAPERSIZE_H
+
+#include <map>
+#include <string>
+
+template<typename T>
+struct Size {
+    Size() : width(0), height(0) {}
+
+    Size(T width_, T height_) : width(width_), height(height_) {}
+
+    bool operator<(const Size &val);
+
+    T width, height;
+};
+
+enum class PaperSizeMetrics {
+    mm,
+    inch
+};
+
+enum class PaperSize {
```

Comment:
Where are these actually used?

Reply:
When I wrote it, i thought it should be used in ComboBox. But now i am using string directly in combobox.